### ISSUE TITLE
Look how it print. What can I do?

### ISSUE THREAD

**Issue by tvanord** [dukelec/mb#11]
![9B9679D0-FA52-4AC3-A825-5FB27AC02BED](https://user-images.githubusercontent.com/65416536/82080585-21fb0e80-96e5-11ea-8d2d-1279ad132790.jpeg)

**Comment by dukelec** [comment]
Same as issue #8, please have a look.

**Comment by tvanord** [comment]
No, it's not the same problem.  I Cran not find such a print image in any of the videos.  I have also tried some tips from it, but without success.

**Comment by dukelec** [comment]
The fault you are experiencing is definitely poor contact. If the relevant treatment in the FAQ does not work, you can contact the seller for a new machine.

**Comment by tvanord** [comment]
Yeah, I'll probably have to send the printer back to Amazon.

Too bad it's no longer available there and I can't get a replacement.

Thanks for trying to solve my problem.

**Comment by tvanord** [comment]
![3F47766C-F12C-42FA-9C13-C905B2B255CE](https://user-images.githubusercontent.com/65416536/82114687-98882280-975e-11ea-87e5-0f6917979ec0.jpeg)
![Uploading 8A73A0E9-4B73-4188-992E-F24DDB40CC63.jpeg…]()
